app/src/config/logger.ts

```diff
@@ -0,0 +1,6 @@
+import { get } from '../utils/env';
+
+export default class Logger {
+    // eslint-disable-next-line yoda
+    public readonly level: string = get('NODE_ENV') === 'development' ? 'debug' : 'info';
```

Comment:
I don't think having the log level depend on another value is smart. I would at least add something so you are able to overwrite the value. Example when you are debugging and you want to get more information so you want to change the log level to verbose or silly.

Reply:
I mean... they could just change it to their appripiate log level there. Is there a better implementation I'm not thinking of? I wouldn't want to use env vars for that purpose as that is mostly taken away from the developers by the provisioning of the environment and the command they run.

Reply:
I would say wrap that as a default value inside another get method that reads in another environment variable.

Reply:
Could you insert a suggestion so I fully understand? (within the formatting of a comment in github)

Reply:
```
public readonly level = get('LOG_LEVEL, get('NODE_ENV', 'development') === 'development' ? 'debug' : 'info');
```

Reply:
Applied